src/Cell.cpp

```diff
@@ -1247,7 +1217,7 @@ void Cell::sectorize(std::vector<Cell*>& subcells) {
   for (int i=0; i < _num_sectors; i++) {
 
     /* Create new Cell clone for this sector Cell */
-    Cell* sector = clone();
+    Cell* sector = clone(false);
```

Comment:
with false option, sector has _region NULL.

Reply:
It has _NULL region, then the sectors are added later. 
The cylinder is included because the sector cell is inside a universe inside the fuel cell, which has the cylinders in its region